lib/ex338_web.ex

```diff
@@ -52,18 +53,18 @@ defmodule Ex338.Web do
       # Use all HTML functionality (forms, tags, etc)
       use Phoenix.HTML
 
-      import Ex338.Router.Helpers
-      import Ex338.ErrorHelpers
-      import Ex338.Gettext
+      import Ex338Web.Router.Helpers
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives

Reply:
Ok